inspect.lua

```diff
@@ -244,7 +244,9 @@ function Inspector:putValue(v)
       elseif self.level >= self.depth then
          puts(buf, '{...}')
       else
-         if self.cycles[t] > 1 then puts(buf, fmt('<%d>', self:getId(t))) end
+         if not self.print_reference then
```

Comment:
Isn't the inside of this block printing references? So it should be:
`if self.print_reference`